### PR TITLE
Add content for demo theme

### PR DESCRIPTION
**Pull request by codesections**
This pull request contains two commits.  The first adds demonstration content that shows off the power of the theme.

The second adds CSS for links in the About page.  Without this CSS, the About page was unstyled (without even default styles, due to the CSS reset).

### CODE REVIEW

File: content/Theme_Overview.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
++++
+title = "Theme Overview"
+date = 2018-07-19
+category = "Real content"
+tags = [
+]
```
Comment (Keats) [inline content/Theme_Overview.md]: the taxonomies need to be updated for 0.4.0

**Comment by Keats** [comment]
I'm wondering if we can add some large pictures from internet, not static in order to test they don't go over boundaries

**Comment by Keats** [comment]
@codesections ping

**Comment by codesections** [comment]
I've addressed the changes suggested above.

My appologies for the delay.

**Comment by Keats** [comment]
Perfect, thanks!
